worker.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -x
+#--- Startup Script for VEDA HTTP Ingest Worker --#
```

Comment:
Can we rename this file to something a bit more descriptive so that we know what it's a script to do